materialized_aggregations.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Materialized Windowed Aggregations
+metaTitle: Materialized Windowed Aggregations
+metaDescription: With Chalk, machine learning teams don’t need to spin up separate pipelines, developers can (cache) materialize aggregations and improve performance of expensive computations with just a single line of code
+description: Cache and materialize a feature aggregation
+published: true
+---
+
+Machine learning teams often build and maintain separate pre-aggregation pipelines that are then ingested--as standalone features--into a feature store to reduce the latency of computing expensive features.
+
+With Chalk, teams don’t need to spin up separate pipelines, developers can materialize aggregations to improve performance all without compromising on flexibility, with just a single line of code:
+
+```py
+from chalk.streams import Windowed, windowed
+
+@features
+class User:
+    id: int
+    transactions: DataFrame[transactions]
+    total_transaction_amount: Windowed[int] = windowed(
+        "10d",
+        "90d",
+        materialization={ # materialize your features in a single line
+            "bucket_duration": "1d", # specify the interval
```

Comment:
I might clarify "interval" here since we have the "window buckets" and the "bucket buckets"